sdk/tests/deqp/functional/gles3/es3fFboRenderTest.js

```diff
@@ -1007,6 +1009,7 @@ goog.scope(function() {
         rrUtil.drawQuad(
             context, texShaderID, [-1.0, -1.0, 0.0], [1.0, 1.0, 0.0]
         );
+        context.disable(gl.BLEND);
```

Comment:
Thanks to point out this. It will not affect the following drawings and other test cases and I think I should keep consistent with C++ code.
https://android.googlesource.com/platform/external/deqp/+/master/modules/gles3/functional/es3fFboRenderTest.cpp#688